CuisineRecipe/CuisineRecipe/Helper/StarRating.swift

```diff
@@ -0,0 +1,26 @@
+//
+//  StarRating.swift
+//  CuisineRecipe
+//
+//  Created by mac on 5/4/19.
+//  Copyright © 2019 sunasterisk. All rights reserved.
+//
+
+final class StarRating: UIStackView {
+    private var ratingButtons = [UIButton]()
+    private let starSize: CGFloat = 25.0
+    
+    func setupButtons(starCount: Int) {
+        for _ in 0..<starCount {
```

Comment:
Thế trường hợp số thập phân thì sao em?

Reply:
API chỉ trả về kiểu int thôi ạ
https://help.yummly.com/hc/en-us/articles/204745224-Rating-and-Reviewing-Recipes